modules/registry-accessing-directly.adoc

```diff
@@ -80,9 +90,17 @@ $ podman pull name.io/image
 The project name must appear in this pull specification for {product-title} to
 correctly place and later access the image in the registry:
 +
+ifdef::openshift-enterprise,openshift-origin[]
 ----
 $ podman tag name.io/image image-registry.openshift-image-registry.svc:5000/openshift/image
 ----
+endif::[
+ifdef::openshift-dedicated[]
+----
+$ podman tag name.io/image $(oc whoami -t) $(oc -n openshift-image-registry get route default-route -o jsonpath='{.spec.host}')/openshift/image
+----
+endif::[]
+]
```

Comment:
```suggestion
```

Reply:
Thanks @jewzaam I couldn't figure out what was holding it up. Technically the build should have failed for that.... :P

I updated the preview build as well: 
http://file.rdu.redhat.com/~antaylor/093019/srep-1943/srep-1943/registry/accessing-the-registry.html